controllers/etcdpeer_controller.go

```diff
@@ -1,3 +1,5 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
ignore all this.. need to do some IDE config rejigging

Reply:
Yeah there was a comment elsewhere (I forget where) about putting copyright or licence boilerplate at the top of files. IANAL but I think it's entirely redundant with a top-level `LICENSE` file like we already have. 🤷‍♂